60-steam-input.rules

```diff
@@ -112,3 +112,12 @@ KERNEL=="hidraw*", KERNELS=="*0955:7214*", MODE="0660", TAG+="uaccess"
 
 # Astro C40
 KERNEL=="hidraw*", ATTRS{idVendor}=="9886", ATTRS{idProduct}=="0025", MODE="0660", TAG+="uaccess"
+
+# Saitek Saitek Pro Flight Yoke
+KERNEL=="hidraw*", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0bac", MODE="0660", TAG+="uaccess"
+KERNEL=="input", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0bac", MODE="0660", TAG+="uaccess"
+KERNEL=="usb", ATTRS{idVendor}=="06a3", ATTRS{idProduct}=="0bac", MODE="0660", TAG+="uaccess"
```

Comment:
It's very unusual to use raw USB to communicate with an input device: normally games and libraries use either evdev (the `input` device) or raw HID (the `hidraw*` device). Also, `KERNEL=="usb"` means `/dev/usb`, which is a directory, not a device. So I don't think this rule is doing anything for you.